LifeSpace/Debug/LogType.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  LogType.swift
+//  LifeSpace
+//
+//  Created by Vishnu Ravi on 11/8/24.
+//
+
+import OSLog
+
+
+enum LogType: String, CaseIterable, Identifiable {
```

Comment:
Could be interesting to explore if it makes sense to conform `OSLogEntryLog.Level` to [`OptionSet`](https://developer.apple.com/documentation/swift/optionset) to utilize some of this here.